backend/src/server/routes/v1/admin-router.ts

```diff
@@ -211,6 +211,27 @@ export const registerAdminRouter = async (server: FastifyZodProvider) => {
     }
   });
 
+  server.route({
+    method: "PATCH",
+    url: "/user-management/users/grant-admin-access",
```

Comment:
let's try to stick to REST conventions. thoughts on having this as  `PATCH /user-management/users/:userId/admin-access` instead?